From 8bd8ba893d25a4d3f847e982de13f180b716b090 Mon Sep 17 00:00:00 2001
Message-Id: <8bd8ba893d25a4d3f847e982de13f180b716b090@dist-git>
From: Erik Skultety <eskultet@redhat.com>
Date: Tue, 2 Aug 2016 15:20:50 +0200
Subject: [PATCH] rpc: virnetserver: Rename ClientSetProcessingControls to
 ClientSetLimits

The original naming was just a leftover that should have been fixed in commit
8b1f0469.

Signed-off-by: Erik Skultety <eskultet@redhat.com>
(cherry picked from commit 2e5417bc466b7a12d72136cdd5f406301ec101d0)

https://bugzilla.redhat.com/show_bug.cgi?id=1357776
Signed-off-by: Erik Skultety <eskultet@redhat.com>
---
 daemon/admin_server.c  | 4 ++--
 src/rpc/virnetserver.c | 6 +++---
 src/rpc/virnetserver.h | 6 +++---
 3 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/daemon/admin_server.c b/daemon/admin_server.c
index 5bbf229..d8a3323 100644
--- a/daemon/admin_server.c
+++ b/daemon/admin_server.c
@@ -382,8 +382,8 @@ adminServerSetClientLimits(virNetServerPtr srv,
                                    VIR_SERVER_CLIENTS_UNAUTH_MAX)))
         maxClientsUnauth = param->value.ui;
 
-    if (virNetServerSetClientProcessingControls(srv, maxClients,
-                                                maxClientsUnauth) < 0)
+    if (virNetServerSetClientLimits(srv, maxClients,
+                                    maxClientsUnauth) < 0)
         return -1;
 
     return 0;
diff --git a/src/rpc/virnetserver.c b/src/rpc/virnetserver.c
index 8c8af97..13b5bd3 100644
--- a/src/rpc/virnetserver.c
+++ b/src/rpc/virnetserver.c
@@ -1046,9 +1046,9 @@ virNetServerGetClient(virNetServerPtr srv,
 }
 
 int
-virNetServerSetClientProcessingControls(virNetServerPtr srv,
-                                        long long int maxClients,
-                                        long long int maxClientsUnauth)
+virNetServerSetClientLimits(virNetServerPtr srv,
+                            long long int maxClients,
+                            long long int maxClientsUnauth)
 {
     int ret = -1;
     size_t max, max_unauth;
diff --git a/src/rpc/virnetserver.h b/src/rpc/virnetserver.h
index 38107b4..151bac8 100644
--- a/src/rpc/virnetserver.h
+++ b/src/rpc/virnetserver.h
@@ -116,8 +116,8 @@ size_t virNetServerGetCurrentClients(virNetServerPtr srv);
 size_t virNetServerGetMaxUnauthClients(virNetServerPtr srv);
 size_t virNetServerGetCurrentUnauthClients(virNetServerPtr srv);
 
-int virNetServerSetClientProcessingControls(virNetServerPtr srv,
-                                            long long int maxClients,
-                                            long long int maxClientsUnauth);
+int virNetServerSetClientLimits(virNetServerPtr srv,
+                                long long int maxClients,
+                                long long int maxClientsUnauth);
 
 #endif /* __VIR_NET_SERVER_H__ */
-- 
2.9.2

